static/src/javascripts/projects/common/modules/discussion/comment-box.js

```diff
@@ -145,6 +146,25 @@ CommentBox.prototype.prerender = function() {
     }
 };
 
+CommentBox.prototype.showOnboarding = function(e) {
+    e.preventDefault();
+
+    // Check if new commenter as they may have already commented on this article
+    if (this.hasState('onboarding-visible') || !this.options.newCommenter) {
```

Comment:
To expand on this, once they have commented we set newCommenter to false (see ln155 just below) but we don't rebind the events. Maybe we should? I wasn't sure about rebinding outside of the ready function.